wgengine/capture/capture.go

```diff
@@ -72,6 +72,10 @@ const (
 	// SynthesizedToPeer indicates the packet was generated from within tailscaled,
 	// and is being routed to a remote Wireguard peer.
 	SynthesizedToPeer Path = 3
+	// SNAT indicates the packet went through a SNAT operation.
```

Comment:
I think instead of duplicating the packet in the PCAP stream we should annotate the existing `FromLocal` path with the translated before/after addresses.

To avoid a comment storm I'll hold off on capture-related comments till weve got the NAT stuff hunkydory.